### ISSUE TITLE
Some stacked bars are stacking on, others are overlapping

### ISSUE THREAD

**Issue by JiRadu** [apexcharts/apexcharts.js#1173]
## Codepen

https://codesandbox.io/s/problem-lnmh3?fontsize=14&hidenavigation=1&theme=dark

## Explanation

- What is the behavior you expect?
I expect all the bars that have the same timestamp to be stacked against each other.
- What is happening instead?

The step in the timeseries is 1h, trimmed to hour ( it is 1:00:00.000, 2:00:00.000, and so on) so when plotting, the values for 2 overlapped stacks are ( as far as I know ) exactly the same. However, if you look at 2 examples:
3 Dec, 19:00 -> two stacked bars are well placed
3 Dec, 21:00 -> the two stacked bars are overlapping
- What error message are you getting?
No error message

**Comment by JiRadu** [comment]
Keep in mind that the series is irregular, so as for the two examples:
19 -> the values are for User 2 and 3 and the value is the first and only for them
21 -> User 5 is the first entry, User 9 is the fourth entry in their respective data

**Comment by JiRadu** [comment]
Hello, @junedchhipa 
First of all, I want to say that this is a kind reminder about this issue, no worries
Second, I want to ask you to acknowledge this issue as a valid issue and tell me if there is any context that I might be missing in isolating the bug.
Can it be a problem because it's an irregular series specifically and when the bar size is calculated, it looks on the series based on the index, not value?
If there is anything I can do, please do tell

**Comment by duckyb** [comment]
I also experienced this issue when creating a time-series stacked chart.
I found that the way to avoid it is to place your timestamps in the "cathegories" prop, and to have arrays of values (without timestamps) in the series property.

However this causes some different (smaller) issues with the xaxis labels.

I think that for now. The only way to avoid all bugs is just to disable the "datetime" option and to set and format the date categories manually.
